src/Core/Logger.php

```diff
@@ -67,73 +65,22 @@ class Logger extends BaseObject
 
 	/**
 	 * Sets the default logging handler for Friendica.
-	 * @todo Can be combined with other handlers too if necessary, could be configurable.
 	 *
 	 * @param LoggerInterface $logger The Logger instance of this Application
-	 *
-	 * @throws InternalServerErrorException if the logger factory is incompatible to this logger
 	 */
-	public static function setLogger($logger)
+	public static function init($logger)
```

Comment:
Please add the `LoggerInterface` type-hint.
```suggestion
	public static function init(LoggerInterface $logger)
```